src/website/views/javascript/tooltip.pug

```diff
@@ -0,0 +1,69 @@
+---
+title: Tooltip
+---
+
+p.-text
+  | Tooltips provide text labels that display when users hover or focus on an element.
+
+h3 Base
+p.-text
+  | To render a tooltip, apply the <code>data-tooltip</code> attribute to an element. Then,
+  | you can initialize it with the chi.tooltip factory method.
+.example.-mb--3
+  .-p--3.-d--flex
+    button(data-tooltip='Your tooltip text on a button').a-btn.-mr--2 Tooltip
+  :code(lang='html')
+    <button id='tootltip-1' class="a-btn" data-tooltip="Your tooltip text on a button">Tooltip</button>
+    <script>chi.tooltip(document.getElementById('tootltip-1'));</script>
+
+h3 Positioning
+p.-text
+  | By default, tooltip's position on top of an element. To alter position, use
+  | the <code>data-position</code> attribute. Valid values are <code>top</code>,
+  | <code>right</code>, <code>bottom</code>, <code>left</code>. You can pass an array
+  | of Elements and initialize all at once.
+.example.-mb--3
+  .-p--3.-d--flex
+    button(data-tooltip='Your top tooltip text').a-btn.-mr--2 Top Tooltip
```

Comment:
@dani-cl-madrid add margin bottom when smaller sizes and remove when goes medium